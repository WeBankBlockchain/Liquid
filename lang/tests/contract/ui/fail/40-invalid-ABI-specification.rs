use liquid_lang as liquid;

#[liquid::interface(name = auto)]
mod foo {
    extern "C" {
        fn bar(&self);
    }
}

fn main() {}
