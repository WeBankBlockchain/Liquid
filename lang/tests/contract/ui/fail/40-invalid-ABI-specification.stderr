error: ABI should be specified  as `"liquid"` or empty
 --> $DIR/40-invalid-ABI-specification.rs:5:5
  |
5 |     extern "C" {
  |     ^^^^^^^^^^
